src/components/ItemFilters/ItemFilters.tsx

```diff
@@ -160,7 +131,55 @@ const FiltersContainer = ({
             onSubmit={handleYearSubmit}
           />
         </Box>
-      </Box>
+      </>
+    )
+  }
+
+  // function passed to filterElements map callback for generating renderMultiSelect prop of MultiSelectGroup
+  const renderMultiSelect = (
+    checkboxGroup: FilterCheckboxGroup,
+    appliedFilterData = {}
+  ) => {
+    return (
+      <MultiSelect
```

Comment:
Pass the prop `closeOnBlur` (https://nypl.github.io/nypl-design-system/reservoir/v3/?path=/docs/components-form-elements-multiselectgroup--docs "Close on Blur State" section) so that the `MultiSelects` close when you click outside or click on a different one.